modules/configuring-your-ide.adoc

```diff
@@ -70,7 +70,9 @@ ifdef::cloud-service[]
 [IMPORTANT] 
 ====
 *Disclaimer:* 
-{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench.
+{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench. +
+
```

Comment:
extra plus at the end of line 73?
Can you fix the passive voice?
 ", which should be reviewed prior to use of this sample workbench."
instead, something like ". You should review their licensing terms before you use this sample workbench."

Reply:
Thanks, I've updated.